atlasdb-client-protobufs/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.6'
```

Comment:
Checked this. It doesn't regenerate anything. (Our auto-regeneration code has never worked too. I've got a fix but it also generates UpgradePersistence.java which we may want to delete from the repo)